pages/devel-version.html

```diff
@@ -92,8 +92,8 @@ <h4><a href="#warning">Warning</a></h4>
             <p>Never use the Uyuni Development Version for production systems in any way!</p>
             <p>Uyuni Development Version is updated with each commit to the Git Repository, and it has the following know limitations:</p>
             <ul>
-                <li>It can break at any moment</li>
-                <li>Upating to newer development versions does not work</li>
+                <li>It may break at any moment</li>
```

Comment:
Why 'may' instead of 'can'?

Reply:
In my non-native-English mind, that expressed probability. But you are the native speaker, so I guess "can" is the right wording :-)

Reply:
Both are correct, but 'may' is a little more ambiguous (it means that you have the ability to do something, or are choosing whether or not to do something, or that you have permission to do something), so most tech writers prefer 'can'. It mostly comes down to personal preference, though. I was mostly just curious :wink: